<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-size:20px;
	/*font-family:sans-serif;*/
	font: 15pt &quot;宋体&quot;;
}
QLineEdit
{
border:noine;
border-radius:10px;
}
QPushButton
{
	background-color: rgb(0, 156, 252);
	border:none;
	border-radius:10px;
}
QPushButton:hover{
	
	background-color: rgb(0, 100, 200);
}
#registerbtn
{
	background-color:none;
	font-size:15px;
	color:white;
}
#backLogin
{
	background-color:none;
	font-size:15px;
	color:white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="loginWidget">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginW
{
	background-image: url(:/bkimage/resource/background/bk_login.jpg);
}
#registerW
{
	background-image: url(:/bkimage/resource/background/bk_login.jpg);
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginW">
      <widget class="QLineEdit" name="userName">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>admin</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>admin</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="loginbtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>登  录</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:20pt;</string>
       </property>
       <property name="text">
        <string>学生管理系统</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="registerbtn">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>注册账号</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="registerW">
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
border:noine;
border-radius:10px;
}
QPushButton
{
	background-color: rgb(0, 156, 252);
	border:none;
	border-radius:10px;
}
QPushButton:hover{
	
	background-color: rgb(0, 100, 200);
}
#reg_affirmpasswd
{
	background-color:none;
}</string>
      </property>
      <widget class="QLineEdit" name="reg_userName">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>admin</string>
       </property>
      </widget>
      <widget class="QLabel" name="reg_label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size:20pt;</string>
       </property>
       <property name="text">
        <string>学生管理系统注册</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="reg_passwd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>admin</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="reg_btn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>注   册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backLogin">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:none;</string>
       </property>
       <property name="text">
        <string>返回登录</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="reg_affirmpasswd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>admin</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
